### PR TITLE
postgresql: change plugin id

### PR DESCRIPTION
**Pull request by gabor**
(part of https://github.com/grafana/grafana/issues/72892)

to move the core postgres plugin closer to external plugins, it's plugin id ( `postgres` ) has to change, and become `grafana-*-datasource`. we set the old name (`postgres`) as an alias, so things should work.

scenarios to test:
- create dashboard before-change, open it after-change, it should work
- create alert before-change, open it after-change, it should work
- create provisioned dashboard with old-plugin-id, load after-change, it should work

### CODE REVIEW

**Review by andresmgot — COMMENTED** [review]
looking good so far :+1:  two things you will probably need to change:

 - Cue files auto-generated with the new path
 - pkg/tests/api/plugins/data/expectedListResp.json with the changes in the plugin

**Comment by gabor** [comment]
> looking good so far 👍 two things you will probably need to change:
> 
> * Cue files auto-generated with the new path
> * pkg/tests/api/plugins/data/expectedListResp.json with the changes in the plugin

thanks, fixed.

**Review by andresmgot — APPROVED** [review]
LGTM :+1:

**Review by zoltanbedi — APPROVED** [review]

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -255,7 +255,7 @@
 
 # BI backend code
```
[the remark this answers is not in the document]
Comment (gabor) [reply .github/CODEOWNERS:256]: sorry, will fix it in a separate PR, (i forgot about this comment, and i really need to merge it)

[the remark this answers is not in the document]
Comment (gabor) [reply .github/CODEOWNERS:256]: fix in https://github.com/grafana/grafana/pull/78034

**Comment by torkelo** [comment]
@gabor 
Why do we have this plugin id requirement? can't we change that? changing plugin id can be very disruptive and cause some breaking changes

**Comment by gabor** [comment]
@torkelo , it's basically in the list of steps defined by plugins-platform. i'll send you a link to the todo-list-document directly.

**Pull request merged**